service/storage_service.cc

```diff
@@ -2125,20 +2125,31 @@ future<> storage_service::raft_state_monitor_fiber(raft::server& raft, cdc::gene
     }
 }
 
+std::unordered_set<raft::server_id> storage_service::find_raft_nodes_from_hoeps(std::list<locator::host_id_or_endpoint> hoeps) {
```

Comment:
should take the list through `const &` to avoid the unnecessary copies and allocations

(by the way it's a mistake that we're using `std::list`, but I understand it was already being used by the caller code for some reason, hence we keep using it)